### DISCUSSION TITLE
[Icon Request] PDF and Excel icons

### DISCUSSION THREAD

**Discussion by philc-d** [tailwindlabs/heroicons#533] [Ideas]
Would be nice to have specific file-type icons, e.g. Excel/PDF; the "document" set is slightly ambiguous as to what the file actually is!

**Comment by hyphen81** [comment]
Anyone know of a library with a similar look and feel to heroicons that might have these file-specific icons? I need to find an alternative.

**Reply by stychu** [reply]
@hyphen81 did you manage to find anything good ?

**Reply by stychu** [reply]
https://react-icons.github.io/react-icons/

**Reply by hyphen81** [reply]
I ended up just using font awesome for those specific icons and heroicons for everything else. Don't love that as a solution, but it's better than just a generic "document" icon.
